src/dsp_tools/commands/excel2json/new_lists/make_new_lists.py

```diff
@@ -196,25 +191,10 @@ def _construct_non_duplicate_id_string(row: pd.Series[Any], preferred_language:
 
 
 def _make_serialised_lists(sheet_list: list[ExcelSheet]) -> list[dict[str, Any]]:
-    all_lists = _make_all_lists(sheet_list)
-    if isinstance(all_lists, ListCreationProblem):
-        msg = all_lists.execute_error_protocol()
-        logger.error(msg)
-        raise InputError(msg)
-    return [list_.to_dict() for list_ in all_lists]
-
-
-def _make_all_lists(sheet_list: list[ExcelSheet]) -> list[ListRoot] | ListCreationProblem:
-    good_lists = []
-    problem_lists: list[SheetProblem] = []
+    all_lists = []
     for sheet in sheet_list:
-        if isinstance(new_list := _make_one_list(sheet), ListRoot):
```

Comment:
I previously removed all the unnecessary checks. It is not even possible now to not get a `ListRoot` I forgot to delete this top-level check